src/app/pages/my-request-detail/my-request-detail.component.html

```diff
@@ -55,41 +55,46 @@ <h2>Queries</h2>
     ></mdm-data-query-row>
 
     <h2>Data elements</h2>
-
     <p *ngIf="request.status === 'unsent'">
       You can add more data elements to this request by
       <a routerLink="/browse">browsing</a> or <a routerLink="/search">searching</a> our
       catalogue.
     </p>
 
+    <div class="mdm-my-request__request-elements-footer">
+      <mat-checkbox (change)="onSelectAll($event)" [checked]="selectAllElements"
+        >Select all</mat-checkbox
+      >
+      <button
+        class="ms-4"
+        mat-raised-button
+        color="primary"
+        [disabled]="removeSelectedButtonDisabled"
+        (click)="removeSelected()"
+      >
+        Remove selected ...
```

Comment:
Might be a bug with the "Remove selected..." button, looking at a "submitted" request, I can get in this state:

![image](https://user-images.githubusercontent.com/3219480/216380503-7294366c-06ac-4799-94d0-0e3e1c880897.png)

The individual remove buttons on each row are hidden at least, but the selection area remove button isn't.

Reply:
It doesn't make sense for the button to be visible, or the checkboxes to be visible when a request has been submitted. I have updated the code to remove the check boxes. Since I had done that, I also went ahead and updated the Templates page so that it uses the new data list of elements, which removes the need for a follow-up task.